examples/integration_with_error.py

```diff
@@ -42,51 +44,38 @@
 data /= 0.5 * 0.25  # the dt in the integral for both dims
 data /= sqrt(ndshape(data)["ph1"] * ndshape(data)["ph2"])  # normalization
 # }}}
+data.ft("t2", shift=True)
+dt = data.get_ft_prop("t2","dt")
+# {{{ vector-normalize the FT
+data /= sqrt(ndshape(data)['t2']) * dt
+# }}}
 # {{{ First, run the code that automatically chooses integration bounds
 # and also assigns error
 fl.next("compare manual vs. automatic", legend=True)
-error_pathway = (
-    set(((j, k) for j in range(2) for k in range(4)))
-    - set(excluded_pathways)
-    - set([(signal_pathway["ph1"], signal_pathway["ph2"])])
+error_pathway = (set(((j, k) for j in range(ndshape(data)['ph1']) for k in range(ndshape(data)['ph2'])))
+        - set(excluded_pathways)
+        - set([(signal_pathway['ph1'], signal_pathway['ph2'])])
 )
-error_pathway = [{"ph1": j, "ph2": k} for j, k in error_pathway]
-data = integral_w_errors(data, signal_pathway, error_pathway, fl=fl)
-fl.plot(data, ".", label="auto: real", capsize=6)
-fl.plot(data.imag, ".", label="auto: imaginary", capsize=6)
+error_pathway = [{'ph1':j, 'ph2':k} for j,k in error_pathway]
+s_int, returned_frq_slice = integral_w_errors(data, signal_pathway, error_pathway,
+        fl=fl, return_frq_slice=True)
+fl.plot(s_int, ".", label="fully auto: real", capsize=6)
+fl.plot(s_int.imag, ".", label="fully auto: imaginary", capsize=6)
 # }}}
 logger.debug(strm("check the std after FT", std(data["ph1", 0]["ph2", 0].data.real)))
 # the sqrt on the next line accounts for the var(real)+var(imag)
-std_off_pathway = data["ph1", 0]["ph2", 0].C.run(lambda x: abs(x) ** 2).mean_all_but(
-    "t2"
-).mean("t2").run(sqrt) / sqrt(2)
-logger.debug(strm("check the std after FT -- manually", std_off_pathway))
-logger.debug(
-    strm(
-        "check the std after FT -- manually don't collapse vd",
-        data["ph1", 0]["ph2", 0]
-        .C.run(lambda x: abs(x) ** 2)
-        .mean_all_but(["t2", "vd"])
-        .mean("t2")
-        .run(sqrt)
-        / sqrt(2),
-    )
-)
-dt = data.get_ft_prop("t2","dt")
-data.ft("t2", shift=True)
-# {{{ vector-normalize the FT
-data /= sqrt(ndshape(data)["t2"]) * dt
-# }}}
```

Comment:
this was originally done below, inside the loop, and makes more sense there.

Reply:
A general note/question: whats the difference between integration_w_error.py and check_integration? is it simply the number of scans/repeats? do we really need two examples for that is seems extra to me

Reply:
one is about checking the auto-integration routine, and the other is about checking that we are doing the error propagation correctly.  I renamed the title of the latter (actually independently, b/c I noticed the same thing).